src/Providers/SessionServiceProvider.php

```diff
@@ -70,4 +74,15 @@ private function storePreviousUrlToSession()
 
         $this->session->save();
     }
+
+    private function shouldGarbageCollect()
+    {
+        $lottery = Config::get('session.lottery');
```

Comment:
I think we'd want to add a default for this, so it works as expected if you do not have this config setting